### ISSUE TITLE
incorrect error "$this" can no longer be unset

### ISSUE THREAD

**Issue by WanWizard** [squizlabs/PHP_CodeSniffer#3965]
## Describe the bug
Running ` ~/.composer/vendor/bin/phpcs -p . --standard=PHPCompatibility --runtime-set testVersion 8.5` on my codebase.

I get the error ""$this" can no longer be unset since PHP 7.1.", but this is not what is happening in the code, see the offending line below.

### Code sample
```php
unset(static::$_cached_objects[get_called_class()][static::implode_pk($this)]);
```

### To reproduce
1. Create a php file containing the given line
2. Run phpcs with the given line
3. See error message displayed
```
---------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------
 1730 | ERROR | "$this" can no longer be unset since PHP 7.1.
---------------------------------------------------------------------------------
```

## Expected behavior
Not being shown this error, $this is not being unset on this line, it is used as an argument to a function.

## Versions (please complete the following information)

| | |
|-|-|
| Operating System | [e.g., Windows 10, MacOS 10.15]
| PHP version | [8.5.0]
| PHP_CodeSniffer version | [3.13.5 stable]
| Standard |PHPCompatibility]
| Install type | Composer, local]

## Please confirm:

- [x] I have searched the issue list and am not opening a duplicate issue.
- [x] I confirm that this bug is a bug in PHP_CodeSniffer and not in one of the external standards.
- [ ] I have verified the issue still exists in the `master` branch of PHP_CodeSniffer.

**Comment by jrfnl** [comment]
Please don't open issues in this repo anymore. This repo is abandoned and https://github.com/PHPCSStandards/PHP_CodeSniffer is its successor. See: #3932

Other than that, closing as reported to the wrong repo.

You are reporting an issue coming from the **external standard** PHPCompatibility. The issue should be reported to that project, not here: https://github.com/PHPCompatibility/PHPCompatibility

As an aside, I'm pretty sure what you are reporting has already been fixed in PHPCompatibility and suggest you first test with the 10.0.0-alpha2 release before opening an issue in that repo.

**Comment by WanWizard** [comment]
Sorry about that, learning all the time...
